package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-node-resolve",
-  "version": "1.0.0",
+  "version": "1.0.0-pre.1",
```

Comment:
license field is wrong below (Can't comment on the actual line)

Reply:
oh thanks!  checking

Reply:
Updated that and authors etc:
```
  "license": "BSD-3-Clause",
  "author": "The Polymer Project Authors"
```